### ISSUE TITLE
7th Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#8]
### 진행 전
- 배치(Batch) 1261로 축소, 에폭(Epoch) 1000회 유지, 학습률(η)도 0.005로 유지
### 진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현
- Softmax로 확률 구현
- Adam으로 Optimizer 구현
- 배치사이즈와 학습률을 두 가지 경우로 개별 학습
- 에폭-손실(Loss) 곡선과 알고리즘을 텐서보드로 저장
- 훈련 파일을 ckpt 확장자로 저장 후 시험
### 진행 후
- 에폭-손실 곡선
![9th Learning](https://user-images.githubusercontent.com/47406388/70208437-e7259f80-1770-11ea-8de1-a63f971f41a2.jpg)
- 손실율이 기존 보다 1e-05~2e-05정도 다소 높게 측정됨
- 정확도가(Acc)가 97.55%로 증가, 예전학습에 비해 약 4.69% 개선됨
- 목표치였던 95% 이상을 달성하여 해당 프로젝트 종료함
